frontend/src/libs/components/portal/styles.module.scss

```diff
@@ -0,0 +1,12 @@
+.noOverflow {
+  overflow: hidden;
+}
+
+.portal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1000;
```

Comment:
Move `z-index` also to variables, and I am not sure that we need `1000`

Reply:
What value do we need?

Reply:
Is 100 enough?

Reply:
I believe yes

Reply:
You may name it something like `z-index-level-one`

Reply:
Got it. Thanks